Tests/App.config

```diff
@@ -0,0 +1,9 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<configuration>
+  <appSettings>
+    <add key="WEBSITE_URL" value="http://automationpractice.com/index.php"/>
+    <add key="NUMBER_OF_SECONDS_TO_WAIT" value="15"/>
+    <add key="INDEX_OF_PRODUCT_ZERO" value="0"/>
+    <add key="INDEX_OF_PRODUCT_ONE" value="1"/>
```

Comment:
Why do these need to be in the config? Maybe you can have a const field of the zero/first index? Unless they aren't going to always be the same index, in which case this is fine